src/wtc-modal-view.js

```diff
@@ -150,7 +165,7 @@ class Modal {
    */
   focusFirstElement() {
     // traverse tree down
-    const findFirst = function(parent) {
+    const findFirst = function (parent) {
```

Comment:
We should probably make these arrow functions, to be consistent with the rest of the codebase

Reply:
Not requesting changes, more of a generic comment here.

Reply:
Converted 🤙